src/core/runtime.hpp

```diff
@@ -41,6 +41,15 @@ using SHTimeDiff = decltype(SHClock::now() - SHDuration(0.0));
 #include <emscripten/val.h>
 #endif
 
+#ifdef TRACY_ENABLE
```

Comment:
You don't need the #if since tracy already defines them to nothing if disabled

Reply:
kinda need in this file due to Valgrind

Reply:
also they don't work well with threads so need to disable them by default (fibers)